css/includes/components/_global-menu.scss

```diff
@@ -206,6 +206,14 @@
                   z-index: 1060;
 
                   & + .dropdown-menu {
+                     &.animate__fadeInLeft:before {
```

Comment:
```suggestion
                     &.animate__fadeInLeft::before {
```

Reply:
for your information, and to avoid this small mistake (it works with single colon):
- single colon are for pseudo **selectors** (like `:not()`, `:nth-child()`)
- double colon are for pseudo **elements** (`::before`, `::after` and some more rare)